docs/products/kafka/kafka-connect/howto/opensearch-sink.rst

```diff
@@ -0,0 +1,111 @@
+
+Furthermore you need to collect the following information about the target OpenSearch service upfront:
+
+* ``OS_CONNECTION_URL``: The OpenSearch connection URL
+* ``OS_USERNAME``: The OpenSearch username to connect
+* ``OS_PASSWORD``: The password for the username selected
+* ``TOPIC_LIST``: The list of topics to sink
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    If you're using Aiven for OpenSearch® and Aiven for Apache Kafka® the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup an OpenSearch sink connector with Aiven CLI
+---------------------------------------------------
+
+The following example demonstrates how to setup a OpenSearch sink connector for Apache Kafka using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``opensearch_sink.json``) with the following content:
+
+.. code-block:: json
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class": "io.aiven.kafka.connect.opensearch.OpensearchSinkConnector",
+        "topics": "TOPIC_LIST",
+        "connection.url": "ES_CONNECTION_URL",
+        "connection.username": "ES_USERNAME",
+        "connection.password": "ES_PASSWORD",
+        "type.name": "TYPE_NAME",
+        "tasks.max":"NR_TASKS",
+        "key.ignore": "true",
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``OS_CONNECTION_URL``, ``OS_USERNAME``, ``OS_PASSWORD``: sink OpenSearch parameters collected in the :ref:`prerequisite <connect_opensearch_sink_prereq>` phase. 
```

Comment:
For consistency with other params, we should use parameter names, not our variables - ``connection.username``, etc. 

Here, and in the section for Elasticsearch

Reply:
fixed